toolkit/tools/imagegen/configuration/configuration.go

```diff
@@ -45,6 +45,16 @@ type InstallScript struct {
 	Path string `json:"Path"`
 }
 
+// PackageRepo defines the RPM repo to pull packages from during the installation
+// or after the system is installed. The "Install" option indicates that the provided
+// repository configuration will be saved in the installed system if specified, and only
+// available during the installation process if not
+type PackageRepo struct {
```

Comment:
This should be in its own file `packagerepo.go` with an accompanying `packagerepo_test.go`. It should also have a set of validation/parsing functions (see something like disk.go etc)